src/port.js

```diff
@@ -0,0 +1,20 @@
+function Port(name) {
+    this.name = name;
+    this.ships = [];
+};
+
+Port.prototype.addShip = function (ship) {
+    this.ships.push(ship);
+};
+
+Port.prototype.removeShip = function (ship) {
+    const i = this.ships.indexOf(ship);
```

Comment:
what if you tried to remove a ship which doesn't exist in `this.ships` array?

Reply:
Resolved this by checking ships array for the instance of Ship and returning an error if not found:
```suggestion
    if (this.ships.includes(ship)) {
    const i = this.ships.indexOf(ship);
  	this.ships.splice(i, 1);
  } else {
		throw new Error(`${ship.name} not found at the port!`);
	}
```